rot13/main.py

```diff
@@ -9,39 +9,6 @@
 	autoescape=True)
 
 
-def encode_algo(character):
```

Comment:
While it's generally the right thing to do to use the standard library, it's nice to see you took the time to implement this while learning.